### ISSUE TITLE
Implement login components

### ISSUE THREAD

**Issue by iurimatias** [status-im/StatusQ#525]
figma: https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1023%3A298171

![Screen Shot 2022-01-12 at 3 47 58 PM](https://user-images.githubusercontent.com/176720/149219170-69eb070f-3664-41ab-a43f-d58c23f680c5.png)

![Screen Shot 2022-01-12 at 3 47 32 PM](https://user-images.githubusercontent.com/176720/149219199-6c6e1751-49a5-464e-9cc7-11def76f4878.png)

![Screen Shot 2022-01-12 at 3 47 09 PM](https://user-images.githubusercontent.com/176720/149219202-43278c2e-173d-4c78-a3b3-4d7f9a3a4dab.png)

![Screen Shot 2022-01-12 at 3 46 51 PM](https://user-images.githubusercontent.com/176720/149219205-6e41dbbb-d125-4ba2-9e91-e713db8b4228.png)

**Comment by 0x-r4bbit** [comment]
It doesn't look like there's a lot in here that isn't already supported by StatusQ. Let me elaborate:

![](https://user-images.githubusercontent.com/176720/149219199-6c6e1751-49a5-464e-9cc7-11def76f4878.png)

^ This has a `StatusSelect` component and a `StatusInput` which makes use of its `component` property. Only thing here is that `StatusInput` doesn't support `echoMode`, so this is probably the time where we need to revisit https://github.com/status-im/status-desktop/issues/7574

![](https://user-images.githubusercontent.com/176720/149219202-43278c2e-173d-4c78-a3b3-4d7f9a3a4dab.png)

^ Assuming we're using `StatusSelect` this can also already be implemented.  If not `StatusSelect`, at the very least, we can probably build this using a combination of `StatusPopupMenu` and some control component.

![](https://user-images.githubusercontent.com/176720/149219205-6e41dbbb-d125-4ba2-9e91-e713db8b4228.png)

^ Here it's just `StatusButton`.

There's one thing I see in the bigger screenshot which is not supported yet in StatusQ and that is the keycard pin input.
An issue for that exists here: https://github.com/status-im/StatusQ/issues/524

So in other words, we can either close this issue now, or we close this once status-im/status-desktop#7574 and status-im/StatusQ#524 are done /cc @iurimatias

**Comment by PascalPrecht** [comment]
Closing this due to inactivity.
Also the login screens have by now been implemented in desktop using existing components as mentioned in my previous comments.
